packages/client/src/components/GameBoard/GameBoard.tsx

```diff
@@ -85,13 +89,38 @@ const GameBoard: React.FC = () => {
     }
   }
 
+  const [isFullscreenEnabled, setFullscreenEnabled] = useState<boolean>(false)
   return (
-    <>
+    <FullscreenProvider
+      enabled={isFullscreenEnabled}
+      onChange={(isFullscreenEnabled: boolean) =>
+        setFullscreenEnabled(isFullscreenEnabled)
+      }>
```

Comment:
[Исправление]
Интерфейс `onChange` и `setFullscreenEnabled` совпадает, можно сразу показать, что параметр - функция
```suggestion
      onChange={setFullscreenEnabled}
      >
```